### ISSUE TITLE
Bedrock InvokeFlow Response Parsing

### ISSUE THREAD

**Issue by grahamburgsma** [soto-project/soto#744]
**Describe the bug**
When using `BedrockAgentRuntime.invokeFlow`, I get the error below. Through some debugging, it appears the request is working correctly, I get the response in the expected format, but `EventStreamDecoder` isn't parsing it correctly. 

I did try the latest `bedrock-agent-runtime.json` file from the aws-sdk-go-v2 repository, but it made no difference.

`No such key 'nodeType' at path ''. No value associated with key CodingKeys(stringValue: \"nodeType\", intValue: nil) (\"nodeType\").`

**To Reproduce**
1. Setup a flow in Bedrock
2. Try to invoke it via BedrockAgentRuntime.invokeFlow`

Happy to help diagnose the issue more as I realize this takes a bit to setup in Bedrock in order to test.

**Expected behavior**
Should parse and return the output event.

**Setup (please complete the following information):**
 - OS: macOS 15.1.1
 - Version of Soto: Core (7.2) and generator (7.2.2)

**Additional context**
I'm using the generator as described [here](https://github.com/soto-project/soto-codegenerator/blob/main/README.md) but haven't had a problem with anything else.

**Comment by adam-fowler** [comment]
There is nowhere to report these issues as far as I can tell. For a while I did know someone working on the SDK team who I reported a load of issues to and most of them got fixed. I have one other person I could try. 

To get around this I have a patch process in the code generator which fixes up errors in model files. 

It appears the official swift SDK doesn't trust the required trait in the models and sets everything to optional. Which in my mind is a bit rubbish.
